### DISCUSSION TITLE
website navigation

### DISCUSSION THREAD

**Discussion by jagadishbabu** [Sunbird-RC/community#88] [General]
Dear dev team

i am not able to browse to roadmap easily from either documentation site or main site of sunbirdrc.dev

please point me to where is the roadmap of capabilities planned

**Comment by parthlawate** [comment]
You can find the roadmap here. https://docs.sunbirdrc.dev/engage/product-roadmap . Right now we only have information on the immediate next release.. But we will try to get more information on there as we progress.
